### PR TITLE
[16.0][MIG] account_credit_control: Migration to 16.0

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

File: account_credit_control/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    """Add a settings on the credit control policy to use on the partners,
+    and links to the credit control lines.
+    """
+
+    _inherit = "res.partner"
+
+    credit_policy_id = fields.Many2one(
+        comodel_name="credit.control.policy",
+        string="Credit Control Policy",
+        # domain="[('account_ids', 'in', property_account_receivable_id)]",
```
Comment (astirpe) [inline account_credit_control/models/res_partner.py:35]: @FernandoRomera what's the status of this porting? Still a work in progress?

**Comment by pedrobaeza** [comment]
Please cherry-pick #253

**Comment by victoralmau** [comment]
Please, cherry-pick https://github.com/OCA/credit-control/pull/259

**Comment by FernandoRomera** [comment]
@pedrobaeza #253 Done
@victoralmau #259 Done

**Comment by victoralmau** [comment]
> @pedrobaeza #253 Done @victoralmau #259 Done

Ok thanks, but you should add those commits to the commit history and not add those changes directly to the 16.0 migration commit.

**Comment by FernandoRomera** [comment]
I don't understand, I always read that I only had to have only one "commit" for the migration processes, do I have to wait for this PR to be merged and then make the changes?

**Comment by pedrobaeza** [comment]
Only one commit for the migration that you have made (there are several that adds a lot of commits to get to the module migrated), not when you cherry-pick others work. If not, you are losing the attribution of the others' work, which is basic in open source.

**Comment by FernandoRomera** [comment]
Understood, but I don't know how to do it. If it seems to you I close the PR and if I find how to do it I will open it again.

**Comment by pedrobaeza** [comment]
You can do an interactive rebase and add the "pick" lines before your commits (pre-commit and migration).

**Comment by FernandoRomera** [comment]
Ok, thanks.

**Comment by astirpe** [comment]
@FernandoRomera seems you deleted the branch. If you restore it, I can continue the porting of this module from what you have already done. Is that ok for you?

**Pull request closed without merging**